tools/generate-models/run.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env zsh
+
+if [[ ! -f "Package.swift" ]]; then
+    echo "\`generate-models/run.sh\` must be run in repository root folder: \`./tools/config/generate-models/run.sh\`"; 
+    exit 1;
+fi
+
+MODE_VERIFY="verify"
+MODE_GENERATE="generate"
+MODE="$1" # verify or generate
+
+OUTPUT_FILE="RUMDataModels.swift"
+TARGET_FILE="$(pwd)/Sources/Datadog/RUM/DataModels/$OUTPUT_FILE"
+
+# get git reference
+if [[ $MODE == $MODE_VERIFY ]]; then
+    TARGET_LINE=$(tail -n 1 $TARGET_FILE)
+    SHA_REGEX='[0-9a-f]{5,40}'
+    if [[ $TARGET_LINE =~ $SHA_REGEX ]]; 
+    then
+        GIT_REF=$MATCH
+    else
+        echo "Target SHA could not be read in $TARGET_FILE !"
+        exit 1;
+    fi
+elif [[ $MODE == $MODE_GENERATE ]]; then
+    GIT_REF="master"
+else
+    echo "Invalid command!"
+    echo "Model generation: run.sh generate"
+    echo "Model verification: run.sh verify"
+    exit 1;
+fi
+
+# fetch rum-events-format for given git ref
+SCRIPT_LOC=$(dirname "$0")
+cd $SCRIPT_LOC
+if [[ ! -d "rum-events-format" ]]; then
+    git clone git@github.com:DataDog/rum-events-format.git
+fi
+cd "rum-events-format"
+git fetch origin $GIT_REF
+git checkout FETCH_HEAD
+SHA=$(git rev-parse HEAD)
+cd ..
+
+# install quicktype and generate model
+if [[ ! -a $(npm bin)/quicktype ]]; then
+    npm install DataDog/quicktype
```

Comment:
How do we specify the version of `DataDog/quicktype`? If it always uses the latest, this won't be possible to leverage caching it between CI builds (upgrading won't be possible), no?

Reply:
this line doesn't check version and fetches the latest commit from this repo 
this is hopefully a temporary workaround because we didn't publish this package yet
as i said in slack, if we publish our fork and cache global `npm` modules in CI then we can improve CI time

for published npm packages, the syntax is like this: `"graphql": "^0.11.7",`

Reply:
OK, I added `RUMM-631: Speed up RUM models generation on CI` to the backlog.